integrations/atlassian/jira/webhooks.go

```diff
@@ -225,58 +179,24 @@ func registerWebhook(ctx context.Context, l *zap.Logger, baseURL, token string)
 		},
 	}
 
-	body, err := json.Marshal(r)
+	u := baseURL + "/rest/api/3/webhook"
+	resp, err := common.HTTPPostJSON(ctx, u, "Bearer "+token, req)
```

Comment:
Thinking about this API, maybe use something line gin's [Context.JSON](https://pkg.go.dev/github.com/gin-gonic/gin#Context.JSON)